search-commons/src/main/java/no/unit/nva/search/SearchClient.java

```diff
@@ -95,7 +97,12 @@ private BoolQueryBuilder matchOneOfOrganizationIdsQuery(ViewingScope viewingScop
         for (URI excludedOrganizationId : viewingScope.getExcludedUnits()) {
             queryBuilder.mustNot(matchPhraseQuery(ORGANIZATION_IDS, excludedOrganizationId.toString()));
         }
-        queryBuilder.mustNot(QueryBuilders.matchQuery(STATUS, APPROVED));
+        searchQueryToRemoveApprovedWorkListAndDraftPublications(queryBuilder);
```

Comment:
the method name does not make sense.  `excludeApprovedDoiRequestsAndDoiRequestsForDraftPublications`